Jojoucsd/day3HW/day3.html

```diff
@@ -0,0 +1,153 @@
+          <img src="http://assets.fodors.com/destinations/54476/hotair-balloons-over-vineyards-napa-and-sonoma-california-usa_main.jpg">
+          <div class="container">
+            <div class="carousel-caption">
+              <h1>Napa Valley</h1>
+              <p>Napa Valley the heart of the U.S winery.One of my favorite place to visit. In this page I added jQuary click(). Please click on the paragraphs, and alert window should pop up</p>
+              <p><a class="btn btn-lg btn-primary" href="#" role="button">Click Me</a></p>
+            </div>
+          </div>
+        </div>
+        <div class="item">
+          <img src="https://upload.wikimedia.org/wikipedia/commons/d/d6/Half_Dome_from_Glacier_Point,_Yosemite_NP_-_Diliff.jpg">
+          <div class="container">
+            <div class="carousel-caption">
+              <h1>Yoesmite</h1>
+              <p>In this page, I am trying to use jQuary hide the images by clicking on the button. And click the "Show Images" on the San Francisco page will bring backthe images.</p>
+              <p><a class="btn btn-lg btn-primary" href="#" role="button" id="hide">Click Images to hide</a></p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <a class="left carousel-control" href="#myCarousel" data-slide="prev"><span class="glyphicon glyphicon-chevron-left"></span></a>
+      <a class="right carousel-control" href="#myCarousel" data-slide="next"><span class="glyphicon glyphicon-chevron-right"></span></a>
+    </div>
+
+    <div class="container threeBlocks">
+
+     <div class="row">
+        <div class="col-lg-4">
+          <h2>Block 1</h2>
+          <p>Block 1 cell for future editing using a "col-lg-4 format </p>
+          <p><a class="btn btn-default" href="#" role="button">Mouseover Block &raquo;</a></p>
+        </div>
+        <div class="col-lg-4">
+          <h2>Block 2</h2>
+          <p>Block 2 cell for future editing using a "col-lg-4 format</p>
+          <p><a class="btn btn-default" href="#" role="button">Mouseover Block &raquo;</a></p>
+        </div>
+        <div class="col-lg-4">
+          <h2>Block 3</h2>
+          <p>Block 3 cell for future editing using a "col-lg-4 format</p>
+          <p><a class="btn btn-default" href="#" role="button">Mouseover Block &raquo;</a></p>
+        </div>
+      </div>
+    </div>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+    <script>
+    $(document).ready(function(){
+        $("p").click(function(){
+            alert("You clicked a paragraph.");
+        });
+    });
+    </script>
+    <script type="text/javascript">
+    $(document).ready(function(){
+        $("#hide").click(function(){
+            $("img").hide();
+        });
+        $("#show").click(function(){
+            $("img").show();
+        });
+    })
+    </script>
+    <script>
```

Comment:
you could combine all your `$(document).ready`s
also consider putting them in an external script file